integration-cli/docker_test_vars.go

```diff
@@ -23,6 +26,8 @@ var (
 	containerStoragePath = dockerBasePath + "/containers"
 
 	workingDirectory string
+
+	goVersion string
```

Comment:
I think `daemonGoVersion` is a better name. at first I thought if this is `runtime.Version()` of the cli or not.